### PR TITLE
AI Content Safety - Initial notebook

### PR DESCRIPTION
**Pull request by andreaskopp**
First notebook of the Solution Accelerator for Content Safety. Developed for Suzana Ilic

### CODE REVIEW

**Review by trrwilson — APPROVED** [review]

File: solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb [hunk 1]
```diff
@@ -0,0 +1,1082 @@
...
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Installs\n",
+        "# %pip install python-dotenv openai==0.28.1 opendatasets tenacity tiktiken"
```
Comment (brandom-msft) [inline solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb:58]:
```suggestion
        "%pip install python-dotenv openai==0.28.1 opendatasets tenacity tiktoken"
```

File: solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb [hunk 2]
```diff
@@ -0,0 +1,1082 @@
...
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
```
Comment (brandom-msft) [inline solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb:392]: This section took a bit over 4 minutes to run - might be worth a comment to the user to expect a wait period.

File: solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb [hunk 3]
```diff
@@ -0,0 +1,1082 @@
...
+        "# Add columns for summaries and Content Filer annotations\n",
```
Comment (brandom-msft) [inline solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb:504]:
```suggestion
        "# Add columns for summaries and Content Filter annotations\n",
```

File: solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb [hunk 4]
```diff
@@ -0,0 +1,1082 @@
...
+        },
+        "gather": {
+          "logged": 1698921459598
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Generate Summaries and track Content Severity Annotations\n",
```
Comment (brandom-msft) [inline solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb:490]: I get "NaN" for all values in the table when I run this, is that expected?

File: solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb [hunk 5]
```diff
@@ -0,0 +1,1082 @@
...
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "code",
```
Comment (brandom-msft) [inline solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb:555]: maybe related to the above comment about "NaN", but when I run this section, the output stops after printing:
       KeyError: 'hate'

Then, further below in "Review Severity Annotations", I see the comment that the rest of the notebook won't generate the expected example output if prior steps were filtered. 

All of that makes sense to me except what I need to/can do to alter the filtering so the rest of the sample will produce output table entries with values.

File: solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb [hunk 6]
```diff
@@ -0,0 +1,1082 @@
...
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Understanding the Impact of Your Content Filter Configuration\n",
+        "\n",
+        "You can configure the content filter for user input and model response separately using the following options:\n",
+        "\n",
+        "![image-alt-text](./cf-table.png)\n",
```
Comment (brandom-msft) [inline solution_accelerators/AI_content_safety/1-news-configure-filter-to-be-less-restrictive.ipynb:720]: missing image?

**Review by brandom-msft — CHANGES_REQUESTED** [review]
It looks great! I ran through most of what I could, but I didn't get some of the outputs that I think were expected (could be user error, though 😄) I left comments along the way.

**Review by brandom-msft — APPROVED** [review]

**Pull request merged**